### PR TITLE
Add SDOGE space

### PR DESCRIPTION
**Pull request by sdoge-master**
**Add space for SDOGE**  
  
Etherscan: https://etherscan.io/address/0xb25FEda5512445Aa4e481Add71731cD0cffF18C1
CoinGecko: 
Website: https://sdoge.org/
Twitter: https://twitter.com/sdoge_master

### CODE REVIEW

File: spaces/sdoge/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/sdoge/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

**Review by e-asia — APPROVED** [review]

**Review by e-asia — CHANGES_REQUESTED** [review]
To use the default theme you will want to delete skin.scss from your space folder.

**Comment by sdoge-master** [comment]
> To use the default theme you will want to delete skin.scss from your space folder.

Deleted

File: spaces/sdoge/index.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "key": "sdoge",
+  "name": "SDOGE",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "SDOGE",
+  "defaultView": "core",
+  "address": "0xb25FEda5512445Aa4e481Add71731cD0cffF18C1",
+  "token": "0xb25FEda5512445Aa4e481Add71731cD0cffF18C1",
```
Comment (e-asia) [inline spaces/sdoge/index.json:9]: Something is wrong with the token, the call to query decimals() is failing. Is this a standard ERC20 token?

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/sdoge/index.json (hunk 2 above)
Comment (sdoge-master) [reply spaces/sdoge/index.json:9]: I've tested, no problem.

Comment (sdoge-master) [reply spaces/sdoge/index.json:9]: FYI, forked from SUSHI.

**Review by e-asia — APPROVED** [review]

**Pull request merged**